vault-consul-acl/intro.md

```diff
@@ -1,17 +1,22 @@
 # Secure Consul Agent Communication with ACL
 
-In this hands-on lab, you will deploy a secure Consul datacenter using Vault to generate and manage certificates.
+In this hands-on lab, you will deploy a secure Consul
+datacenter using Vault to generate and manage certificates.
 
-The lab will guide you through the steps necessary to deploy Consul with TLS encryption enabled to secure access to the UI, API, CLI, services, and agents.
+The lab will guide you through the steps necessary to
+deploy Consul with TLS encryption enabled to secure access
+to the UI, API, CLI, services, and agents.
 
```

Comment:
```suggestion
The lab guided you through the steps necessary to deploy Consul
with ACL enabled to verify identity of your server nodes and assign them necessary permissions

```

Reply:
I'm not sure "verify identity" is the correct phrasing. Maybe something like "deploy a Consul datacenter with ACLs enabled to secure access to the cluster data and agents". Cluster data meaning the catalog. Agents meaning accessing (managing) them with the UI/CLI/API.